plugins/cisco_umbrella_destinations/plugin.spec.yaml

```
@@ -0,0 +1,406 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: cisco_umbrella_destinations
+title: Cisco Umbrella Destinations
+version: 1.0.0
+supported_versions: ["2022-31-01"]
```

ISO8601 format is YYYY-MM-DD
```suggestion
supported_versions: ["2022-01-31"]
```